tests/sentry_plugins/opgsenie/test_plugin.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+
+import responses
+import six
+
+from exam import fixture
+from sentry.models import Rule
+from sentry.plugins.base import Notification
+from sentry.testutils import PluginTestCase
+from sentry.utils import json
+
+from sentry_plugins.opsgenie.plugin import OpsGeniePlugin
+
+INVALID_METHOD = (
```

Comment:
are you going to add a test that uses this?

Reply:
@MeredithAnya No! Good catch :)